css/style.css

```diff
@@ -16,4 +17,182 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header styles */
+ .header {
+     display: flex;
+     position: fixed;
+     background-color: white;
+     padding-bottom: 5px;
+     width: 100%;
+     border-bottom: 2px solid rgb(234, 235, 236);
+     ;
+ }
 
+ .header-img {
+     padding-top: 12px;
+     padding-left: 275px;
+     padding-right: 185px;
+ }
+
+ /* Navigation styles */
+ .links .active a {
+     font-weight: 500;
+     font-size: 17px;
+     text-decoration: none;
+     color: rgb(76, 80, 88);
+     }
+
+ a {
+    text-decoration: none;
+    font-size: 17px;
+    color: rgb(131, 137, 148);
+ }
+
+ .navigation ul {
+     display: flex;
+     flex-direction: row;
+     list-style: none;
+     padding-top: 10px;
+ }
+
+ .navigation li {
+     display: flex;
+     padding-right: 30px;
+     color: darkgrey;
+ }
+
+ .navigation .links a:hover {
+     color: rgb(32, 27, 27);
+ }
+ 
+ /* Introductory section styles */
+.karma-introduction {
+    background-image: url(../img/first-background.jpg);
+    background-position: center bottom;
+    padding: 250px;
+    text-align: center;
+}
+
+.karma-introduction h1 {
+    color: white;
+    font-weight: 300;
+    font-size: 50px;
+    margin: 10px;
+}
+
+.karma-introduction h2 {
+    color: white;
+    font-weight: 300;
+    font-size: 25px;
+    margin: 10px;
+}
+
+.karma-introduction button {
+    background-color: rgb(241,90,41);
+    font-size: 17px;
+    color: white;
+    border: none;
+    padding: 16px 32px;
+    border-radius: 4px;
+    margin: 10px;
+}
+
+.karma-introduction button:hover {
+    background-color: rgb(207, 69, 6);
+}
+
+/* About section styles */
+.about {
+    text-align: center;
+    padding-top: 100px;
+    padding-bottom: 100px;
+    padding-left: 200px;
+    padding-right: 200px;
+}
+
+.about h1 {
+    font-weight: 300;
+    font-size: 45px;  
+    padding-bottom: 25px;
+}
+
+.about img {
+    padding-bottom: 25px;
+}
+
+.about ul {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
```

Comment:
`flex-direction` is defaulted to row, so you don't need to include it - but you can if it makes things more clear.